cli-output/src/display.rs

```diff
@@ -133,7 +133,7 @@ fn format_account_mode(message: &Message, index: usize) -> String {
         } else {
             "-"
         },
-        if message.is_writable(index) {
+        if message.is_writable(index, /*demote_sysvar_write_locks=*/ true) {
```

Comment:
Note: One of the 2 hard-coded values. This one is invoked from `Display` implementation (and maybe other places). So having a hard coded value somewhere is inevitable.